--- gpart-0.1h/src/gm_ntfs.h.org	2004-12-13 13:13:56.114967688 +0000
+++ gpart-0.1h/src/gm_ntfs.h	2004-12-13 13:14:14.776130760 +0000
@@ -29,7 +29,7 @@
 /* 'NTFS' in little endian */
 #define NTFS_SUPER_MAGIC	0x5346544E
 
-#if defined(i386) || defined(__i386__) || defined(__alpha__) || defined(__powerpc__)
+#if defined(i386) || defined(__i386__) || defined(__alpha__) || defined(__powerpc__) || defined(__x86_64__)
 
 /* unsigned integral types */
 #ifndef NTFS_INTEGRAL_TYPES
