diff -uNr gpart-0.1h.old/src/gm_ntfs.h gpart-0.1h/src/gm_ntfs.h
--- gpart-0.1h.old/src/gm_ntfs.h	Mon Jan 29 20:33:58 2001
+++ gpart-0.1h/src/gm_ntfs.h	Sat May 18 21:09:43 2002
@@ -29,7 +29,7 @@
 /* 'NTFS' in little endian */
 #define NTFS_SUPER_MAGIC	0x5346544E
 
-#if defined(i386) || defined(__i386__) || defined(__alpha__)
+#if defined(i386) || defined(__i386__) || defined(__alpha__) || defined(__powerpc__)
 
 /* unsigned integral types */
 #ifndef NTFS_INTEGRAL_TYPES
@@ -39,7 +39,7 @@
 typedef unsigned int		ntfs_u32;
 typedef s64_t			ntfs_u64;
 #endif /* NTFS_INTEGRAL_TYPES */
-#endif /* defined(i386) || defined(__i386__) || defined(__alpha__) */
+#endif /* defined(i386) || defined(__i386__) || defined(__alpha__) || defined(__powerpc__)*/
 
 
 /* Macros reading unsigned integers from a byte pointer */
