diff -Nur gpart-0.1h.org/src/disku.c gpart-0.1h/src/disku.c
--- gpart-0.1h.org/src/disku.c	Wed Feb  7 19:04:07 2001
+++ gpart-0.1h/src/disku.c	Sun Mar 16 11:20:39 2003
@@ -19,6 +19,8 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/ioctl.h>
+#include <errno.h>
+
 #include "gpart.h"
 
 #if defined(__linux__)
diff -Nur gpart-0.1h.org/src/l64seek.c gpart-0.1h/src/l64seek.c
--- gpart-0.1h.org/src/l64seek.c	Wed Dec 13 22:54:11 2000
+++ gpart-0.1h/src/l64seek.c	Sun Mar 16 11:23:03 2003
@@ -14,6 +14,7 @@
  *
  */
 
+#include <errno.h>
 #include "l64seek.h"
 
 
